src/plugins/saved_objects_management/public/plugin.ts

```diff
@@ -117,21 +118,72 @@ export class SavedObjectsManagementPlugin
       });
     }
 
-    const opensearchDashboardsSection = management.sections.section.opensearchDashboards;
-    opensearchDashboardsSection.registerApp({
-      id: 'objects',
-      title: i18n.translate('savedObjectsManagement.managementSectionLabel', {
-        defaultMessage: 'Saved Objects',
+    const mountWrapper = ({
+      title,
+      allowedObjectTypes,
+    }: {
+      title: string;
+      allowedObjectTypes?: string[];
+    }) => async (mountParams: AppMountParameters) => {
+      const { mountManagementSection } = await import('./management_section');
+      return mountManagementSection({
+        core,
+        serviceRegistry: this.serviceRegistry,
+        mountParams,
+        title,
+        allowedObjectTypes,
+      });
+    };
+
+    /**
+     * Register saved search & saved query here
```

Comment:
Nit: seems this comment is not relevant here.

Reply:
Done for this one.